package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "erc-checker",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
This is more than a bugfix release, so please change the version to `1.3`.

Reply:
All right